README.md

```diff
@@ -1,3 +1,44 @@
+# add_team_usage_auth
+
+For testing purposes, create a 'New OAuth App'. Go to this [page](https://github.com/settings/developers) where you can create the app. Note down the Client ID and Client secret.
+
+- Set the application name to anything.
```

Comment:
It is worth giving a sensical example - e.g github-auth-app

Reply:
Added this - although apps can't start with github or git.